example-apps/stripe-webshop/.env.ci

```
@@ -0,0 +1,11 @@
+
+# Database URL for PGlite
+DATABASE_URL="./dev.pglite"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)